cli/src/commands/init.rs

```diff
@@ -90,9 +119,86 @@ fn prompt_group() -> io::Result<Option<String>> {
         return Ok(None);
     }
 
-    let group: String = Input::with_theme(&ColorfulTheme::default())
-        .with_prompt("Project Group (default: none)")
-        .interact_text()?;
+    let group: String =
+        Input::new().with_prompt("Project Group (default: none)").interact_text()?;
 
     Ok(Some(group))
 }
+
+/// Interactively ask for missing lockfile information.
+fn prompt_lockfile(
+    cli_lockfile: Option<&String>,
+    cli_lockfile_type: Option<&String>,
+) -> io::Result<(String, Option<String>)> {
```

Comment:
The returned option is always `Some(_)`. So it might make sense to refactor the `Option` away.

```suggestion
fn prompt_lockfile(
    cli_lockfile: Option<&String>,
    cli_lockfile_type: Option<&String>,
) -> io::Result<(String, String)> {
```

Not really a big deal either way though, so I'll approve anyways.

Reply:
Maintainable code is always a big deal, good catch.